lib/processors/versionInfoGenerator.js

```diff
@@ -14,6 +16,335 @@ function getTimestamp() {
 	return year + month + day + hours + minutes;
```

Comment:
I don't remember what we said about timestamps. UTC or not?

Reply:
handled in separate PR
https://github.com/SAP/ui5-builder/pull/348